client/state/data-layer/wpcom/timezones/index.js

```diff
@@ -43,21 +43,24 @@ export const fromApi = ( { manual_utc_offsets, timezones, timezones_by_continent
 /*
  * Start a request to WordPress.com server to get the timezones data
  */
-export const fetchTimezones = ( { dispatch }, action ) =>
-	dispatch(
-		http(
-			{
-				method: 'GET',
-				path: '/timezones',
-				apiNamespace: 'wpcom/v2',
-			},
-			action
-		)
+export const fetchTimezones = action =>
+	http(
+		{
+			method: 'GET',
+			path: '/timezones',
+			apiNamespace: 'wpcom/v2',
+		},
+		action
 	);
 
-export const addTimezones = ( { dispatch }, action, data ) =>
-	dispatch( timezonesReceive( fromApi( data ) ) );
+export const addTimezones = ( action, data ) => timezonesReceive( data );
 
 export default {
-	[ TIMEZONES_REQUEST ]: [ dispatchRequest( fetchTimezones, addTimezones ) ],
+	[ TIMEZONES_REQUEST ]: [
+		dispatchRequestEx( {
+			fetch: fetchTimezones,
+			onSuccess: addTimezones,
```

Comment:
Not having a specific `onError` handler generates a console warning:

![](https://cldup.com/dVrxH4mwyc.png)

Perhaps we could define a `onError: noop` there.

Reply:
nooped it in f9e7e96